azure-pipelines.yml

```
@@ -239,7 +272,38 @@ jobs:
         rust_version: 1.31.0
     - bash: |
         sudo apt-get update &&
-        sudo apt-get -y install libsqlite3-dev libpq-dev libmysqlclient-dev
+        sudo apt-get install wget &&
```

This misses libpq and mysqlclient
```suggestion
        sudo apt-get install wget libpq-dev libmysqlclient-dev &&
```